### PR TITLE
[DOCS] Create About GX Cloud Content

### PR DESCRIPTION
**Pull request by kwcanuck**
[DSB-366](https://greatexpectations.atlassian.net/browse/DSB-366) and [DSB-351](https://greatexpectations.atlassian.net/browse/DSB-351) request the addition of an overview topic that discusses the features and benefits of GX Cloud, the typical workflow, and supported browsers. This PR implements the addition of this content.

## Definition of done
- [X] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [X] _Appropriate_ tests and docs have been updated

[DSB-366]: https://greatexpectations.atlassian.net/browse/DSB-366?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DSB-351]: https://greatexpectations.atlassian.net/browse/DSB-351?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ghost** [comment]
<details open>
<summary><b>👇 <a href="https://app.codesee.io/r/reviews?pr=8834&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations">Click on the image</a> for a new way to code review</b>
</summary>

####

[Review these changes using an interactive CodeSee Map](https://app.codesee.io/r/reviews?pr=8834&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations)

#### Legend
CodeSee Map legend
</details>

File: docs/docusaurus/docs/cloud/about_gx.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+GX Cloud builds on the innovative features of GX open source software (OSS). Its easy-to-use web interface lets you quickly validate your data without creating and running complex Python code. With GX Cloud you can work collaboratively with your teammates to define and test reusable data queries that alert you to changes in your data.
+
+Here are just a few of the benefits you'll get with GX Cloud: 
+
+- It's web-based and browser and platform-independent.
+
+- A hosted environment and intuitive interface makes it faster to set up and run your Data Validations.
+
+- Unique features such as Validation run histories, Data Asset metrics, and more.
```
Comment (talagluck) [inline docs/docusaurus/docs/cloud/about_gx.md]:
```suggestion
- Features such as Validation run histories, Data Asset metrics, and more.
```

File: docs/docusaurus/docs/cloud/about_gx.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+- Eliminates the need to create, run, and maintain complex code for your data queries.
+
+## Workflow optimization
+
+GX Cloud is intended to get you started faster than GX OSS and to simplify your Data Validation workflows. Here's how it's done:
```
Comment (talagluck) [inline docs/docusaurus/docs/cloud/about_gx.md]:
```suggestion
GX Cloud is intended to help you get started faster than GX OSS and to simplify your Data Validation workflows. Here's how it's done:
```

**Review by talagluck — COMMENTED** [review]
Looks good overall. A couple of minor suggestions. I'll leave to Erik or Erin to approve.

**Comment by erik-hencier** [comment]
Same, it looks good to me.

**Review by erik-hencier — APPROVED** [review]
Good to go.

File: docs/docusaurus/docs/cloud/about_gx.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+description: Learn more about the features and benefits of GX Cloud.
+toc_min_heading_level: 2
+toc_max_heading_level: 2
+---
+
+GX Cloud builds on the innovative features of GX open source software (OSS). Its easy-to-use web interface lets you quickly validate your data without creating and running complex Python code. With GX Cloud you can work collaboratively with your teammates to define and test reusable data queries that alert you to changes in your data.
+
+Here are just a few of the benefits you'll get with GX Cloud: 
+
+- It's web-based and browser and platform-independent.
```
Comment (Erin-GX) [inline docs/docusaurus/docs/cloud/about_gx.md]:
```suggestion
- It's web-based and browser- and platform-independent.
```

File: docs/docusaurus/docs/cloud/about_gx.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+- A hosted environment and intuitive interface makes it faster to set up and run your Data Validations.
```
Comment (Erin-GX) [inline docs/docusaurus/docs/cloud/about_gx.md]:
```suggestion
- A hosted environment and intuitive interface makes it faster to set up and run your data Validations.
```

File: docs/docusaurus/docs/cloud/about_gx.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+- Features such as Validation run histories, Data Asset metrics, and more.
+
+- Eliminates the need to create, run, and maintain complex code for your data queries.
```
Comment (Erin-GX) [inline docs/docusaurus/docs/cloud/about_gx.md]:
```suggestion
- No more need to create, run, and maintain complex code for your data queries.
```

File: docs/docusaurus/docs/cloud/about_gx.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+## Workflow optimization
+
+GX Cloud is intended to help you get started faster than GX OSS and to simplify your Data Validation workflows. Here's how it's done:
```
Comment (Erin-GX) [inline docs/docusaurus/docs/cloud/about_gx.md]:
```suggestion
GX Cloud helps you get started faster and simplifies your data Validation workflows. Here's how:
```

File: docs/docusaurus/docs/cloud/about_gx.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+
+- [Mozilla Firefox](https://www.mozilla.org/en-US/firefox/) — the latest version is fully supported
+
+- [Apple Safari](https://www.apple.com/safari/) — the latest version is fully supported
+
+- [Microsoft Edge](https://www.microsoft.com/en-us/edge?ep=82&form=MA13KI&es=24) — the latest version is fully supported
+
+### Browser session duration
+
+A session is the period of time that you’re signed in to your GX Cloud account from a browser. If you close the browser, your session ends you're signed out. You'll need to sign in again to access GX Cloud.
```
Comment (Erin-GX) [inline docs/docusaurus/docs/cloud/about_gx.md]:
```suggestion
A session is the period of time that you’re signed in to your GX Cloud account from a browser. If you close the browser, your session ends and you're signed out. You'll need to sign in again to access GX Cloud.
```

**Review by Erin-GX — APPROVED** [review]
I'll have more extensive edits after I create the "Why Cloud" doc, in the meantime just commented a couple things that jumped out at me

File: docs/docusaurus/docs/cloud/about_gx.md [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+
+The following table lists the tasks that you complete in a typical GX Cloud workflow. A brief description of the task and links to the relevant topics are provided.
+
+| Task                                     | Description                                                        | Related topic               |
+|------------------------------------------|--------------------------------------------------------------------|-----------------------------|
+| Set up GX Cloud                          | Set up your environment to get the most out of GX Cloud. Do it once, and you don't need to do it again.           | [Set up GX Cloud](/docs/cloud/set_up_gx_cloud)                         | 
+| Connect to a Data Asset on a Data Source | The Data Asset contains the data you want to examine and Validate. | [Create a Data Asset](/docs/cloud/data_assets/manage_data_assets#create-a-data-asset)                           |
```
Comment (HaebichanGX) [inline docs/docusaurus/docs/cloud/about_gx.md]: nit: can we change "Validate" to "validate"?

**Review by HaebichanGX — APPROVED** [review]
LGTM, just have a small nit that I pointed out in a comment

File: docs/docusaurus/docs/cloud/about_gx.md (hunk 8 above)
Comment (kwcanuck) [inline docs/docusaurus/docs/cloud/about_gx.md]:
```suggestion
| Connect to a Data Asset on a Data Source | The Data Asset contains the data you want to examine and validate. | [Create a Data Asset](/docs/cloud/data_assets/manage_data_assets#create-a-data-asset)                           | 
```

Comment (kwcanuck) [reply docs/docusaurus/docs/cloud/about_gx.md]: @HaebichanGX done! Thanks.

**Pull request merged**